src/components/core/InlineComponent.js

```diff
@@ -4,18 +4,19 @@ Job: nothing yet, but adding a isInline parameter to an inline component
 
 Knows: parent dimensions
 
-*/
-export default function InlineComponent( Base = class {} ) {
+ */
+export default function InlineComponent( Base = class {
+} ) {
```

Comment:
That's crappy indeed.

We could simply stay with:
```javascript
export default function InlineComponent( Base = class {} ) {
```
But this would be an exception on a syntaxic rule that tries to break braces of a class definition. That not bat itself, but IDE tends to handle very poorly syntaxic exception. This means IDE's will try to "fix it" as soon as we ask a "reformat/fix indent/etc"... So we will have to be cautious and meticulous.

Another option will be to remove ` = class{}` in those functions. 
I tried and nothing seems to be broken. 

I wonder if it really make sense to construct a InlineComponent without any provided Base. 
Is Base really an optional parameter?

If `Base = class{}` was only added to provide "in code documentation" we could simply add some jsDoc or rename `Base` to `BaseClass`

Reply:
I've just remove them. I going to perform some more checks on it